### PR TITLE
Dont break fs 2

### PR DESCRIPTION
**Pull request by damianatorrpm**
continuation on recreated branch from https://github.com/WayfireWM/wayfire/pull/633

### CODE REVIEW

**Comment by damianatorrpm** [comment]
before reviewing => I will need to rebuild OS and test the changes

File: plugins/single_plugins/move.cpp [hunk 1]
```diff
@@ -324,7 +324,21 @@ class wayfire_move : public wf::plugin_interface_t
             return;
         }
 
-        /* Snap the view */
+        /**
+         * Snap the view
```
Comment (ammen99) [inline plugins/single_plugins/move.cpp:328]: Shouldn't you move this comment down below? (line 713)

File: plugins/common/wayfire/plugins/common/move-snap-helper.hpp [hunk 2]
```diff
@@ -140,6 +140,27 @@ class move_snap_helper_t : public wf::custom_data_t
...
 
         view->disconnect_signal("geometry-changed", &view_geometry_changed);
+        /**
+         * Restore the fullscreen state so the window is
+         * not in a weird position afterwards, fullscreen
+         * are by spec the geometry of the output (or span outputs)
+         */
+        if (view->fullscreen)
+        {
+            wf::output_t *output = wf::get_core().get_active_output();
```
Comment (ammen99) [inline plugins/common/wayfire/plugins/common/move-snap-helper.hpp:150]: you should use `view->get_output()`

**Review by ammen99 — COMMENTED** [review]
This one looks good, I have the following to add: ~~http://ix.io/2BHe~~ http://ix.io/2BHf  add it to the pr and then we can merge.

File: plugins/common/wayfire/plugins/common/move-snap-helper.hpp (hunk 2 above)
Comment (damianatorrpm) [reply plugins/common/wayfire/plugins/common/move-snap-helper.hpp:150]: I haven't tested it yet, but I'm not sure what's supposed to happen
If I use the views output and I drag the fullscreen view 
from output A (no expo) to output B (expo active),
I would need to include some calculation for the ws on output B?

Comment (ammen99) [reply plugins/common/wayfire/plugins/common/move-snap-helper.hpp]: if you drag the view to output B then view->get_output() will already point to output B, this is why I said that the view's output shoukd be used.

Comment (ammen99) [reply plugins/common/wayfire/plugins/common/move-snap-helper.hpp]: also don't worry about expo or no expo, the move-snap-helper takes care of that

**Review by ammen99 — APPROVED** [review]

**Comment by ammen99** [comment]
Thanks!

**Pull request merged**